modules/otel-config-multicluster.adoc

```diff
@@ -6,19 +6,107 @@
 [id="gathering-observability-data-from-different-clusters_{context}"]
 = Gathering the observability data from different clusters with the OpenTelemetry Collector
 
-For a multicluster configuration, you can create one OpenTelemetry
-Collector instance in each one of the remote clusters and forward all the telemetry
-data to one OpenTelemetry Collector instance.
+For a multicluster configuration, you can create one OpenTelemetry Collector instance in each one of the remote clusters and then forward all the telemetry data to one OpenTelemetry Collector instance.
```

Comment:
skrthomas, I'd recommend preferring the core OpenShift pages, which use `multicluster`, rather than the aligned product pages: https://docs.openshift.com/search.html?query=multicluster. Also, IBM Style returns only `multicluster` results and none of `multi-cluster` results.

Reply:
Fair enough, the repo goes both ways it seems, and you're right that IBMSG doesn't hyphenate. I'm fine with leaving it as it is.